buildscripts/semantic-convention/generate.sh

```diff
@@ -30,6 +35,12 @@ git fetch origin "$SPEC_VERSION"
 git reset --hard FETCH_HEAD
 cd ${SCRIPT_DIR}
 
+# echo "Help ..."
```

Comment:
Is this some temporary code?

Reply:
This is for maintainers, to get the command line help of the tool itself, if needed.